ebcDockerBuilderLGS.jenkinsfile

```diff
@@ -0,0 +1,139 @@
+properties([
+   parameters([
+      // EBC relevant properties
+      string(name: 'executionId', defaultValue: UUID.randomUUID().toString(), description: 'Unique execution id'),
+      string(name: 'ebcPriority', defaultValue: '200', description: 'EBC Priority'),
+      string(name: 'ebcPlan', defaultValue: 'svl-dockerJenkins-ubuntu20_ppcle.yml', description: 'EBC plan to use when provisioning a Jenkins node'),
+      string(name: 'ebcBranch', defaultValue: "${env.ecosystem_branch}", description: 'Git branch used for ebc code'),
+      string(name: 'scriptOrg', defaultValue: "OpenLiberty", description: 'Git org containing docker build scripts'),
+      string(name: 'command', defaultValue: "make build-sample REGISTRY=cp.stg.icr.io", description: 'Build command to execute on target arch machine, e.g. make build-pipeline-releases'),
+      string(name: 'PIPELINE_OPERATOR_IMAGE', defaultValue: "cp/liberty-getting-started-sample", description: 'namespace to push image to in registry'),
```

Comment:
string(name: 'IMAGE', defaultValue: "cp.stg.icr.io/cp/liberty-getting-started-sample", description: 'namespace to push image to in registry'),